New Changelog: WakaTime is now a GitHub secret scanning partner (#144)
yuhattor/hattori.dev: content/changelog/2023-02-10-wakatime-is-now-a-github-secret-scanning-partner.md

author: "Kevin Duck"
description: " GitHub secret scanning protects users by searching repositories for known types of secrets. By identifying and flagging these secrets, our scans help prevent data leaks and fraud.  We have partnered with WakaTime to scan for their tokens and help secure our mutual users on public repositories. WakaTime tokens allow users to programmatically access their WakaTime code statistics. GitHub will forward access tokens found in public repositories to WakaTime, who will immediately revoke the leaked token and email the token's owner with instructions on next steps. You can read more information about WakaTime tokens here .  GitHub Advanced Security customers can also scan for WakaTime tokens and block them from entering their private and public repositories with push protection .  Learn more about secret scanning  Partner with GitHub on secret scanning  "
coverimage: ""
englishsummary: "  GitHub secret scanning helps prevent data leaks and fraud by searching repositories for known types of secrets and partnering with WakaTime to scan for their tokens, while Advanced Security customers can also block them from entering their repositories."

> summary: "  GitHubの秘密スキャンは、既知の種類の秘密についてリポジトリを検索し、WakaTimeと提携してそのトークンをスキャンすることで、データ漏洩や不正行為の防止に役立ちます。また、Advanced Securityのお客様は、リポジトリへの侵入をブロックすることも可能です。"

Reviewer: ```suggestion
summary: "GitHubのシークレットスキャンは、既知の種類の秘密についてリポジトリを検索し、WakaTime と提携してそのトークンをスキャンすることで、データ漏洩や不正行為の防止に役立ちます。また、Advanced Security のお客様は、リポジトリへの侵入をブロックすることも可能です。"
```

---

<p>GitHub シークレットスキャンは、既知のタイプのシークレットをリポジトリから検索することで、ユーザーを保護します。これらの秘密を特定し、フラグを立てることで、私たちのスキャンはデータ漏洩や不正行為を防ぐのに役立っています。</p>

> <p>私たちは<a href="https://wakatime.com/">WakaTimeと</a>提携し、彼らのトークンをスキャンして、公開リポジトリ上の相互ユーザーの安全を確保する手助けをしています。WakaTimeトークンは、ユーザーがWakaTimeのコード統計にプログラムでアクセスすることを可能にします。GitHubは公開リポジトリで見つかったアクセストークンをWakaTimeに転送し、WakaTimeは流出したトークンを直ちに失効させ、次のステップの指示をトークンの所有者に電子メールで送信します。WakaTimeトークンの詳細については、<a href="https://wakatime.com/developers">こちらを</a>ご覧ください。</p>

Reviewer: ```suggestion
<p>私たちは<a href="https://wakatime.com/">WakaTime</a>と提携し、彼らのトークンをスキャンして、公開リポジトリ上の相互ユーザーの安全を確保する手助けをしています。WakaTimeトークンは、ユーザーがWakaTimeのコード統計にプログラムでアクセスすることを可能にします。GitHubは公開リポジトリで見つかったアクセストークンをWakaTimeに転送し、WakaTimeは流出したトークンを直ちに失効させ、次のステップの指示をトークンの所有者に電子メールで送信します。WakaTimeトークンの詳細については、<a href="https://wakatime.com/developers">こちらを</a>ご覧ください。</p>
```

> <p>GitHub Advanced Securityのお客様は、WakaTimeトークンをスキャンし、<a href="https://github.blog/changelog/2022-04-04-secret-scanning-prevents-secret-leaks-with-protection-on-push/">プッシュ保護で</a>プライベートおよびパブリックリポジトリへのトークンの侵入をブロックすることも可能です。</p>

Reviewer: ```suggestion
<p>GitHub Advanced Security のお客様は、WakaTimeトークンをスキャンし、<a href="https://github.blog/changelog/2022-04-04-secret-scanning-prevents-secret-leaks-with-protection-on-push/">プッシュ保護で</a>プライベートおよびパブリックリポジトリへのトークンの侵入をブロックすることも可能です。</p>
```
